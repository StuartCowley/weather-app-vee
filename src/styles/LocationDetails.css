h1 {
  display: flex;
  justify-content: center;
}

@media (min-width: 992px) {
  h1 {
    display: block;
  }
}
