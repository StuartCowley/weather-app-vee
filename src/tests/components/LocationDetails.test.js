import React from "react";
import { render } from "@testing-library/react";
import LocationDetails from "../../components/LocationDetails";

describe("LocationDetail", () => {
  const validProps = {
    city: "Manchester",
    country: "UK",
  };
  it("renders correctly", () => {
    const { asFragment } = render(
      <LocationDetails city={validProps.city} country={validProps.country} />
    );

    expect(asFragment()).toMatchSnapshot();
  });

  it("renders the correct city and location props", () => {
    const { getByText } = render(
      <LocationDetails city={validProps.city} country={validProps.country} />
    );

    expect(getByText(/Manchester/i)).toBeInstanceOf(HTMLHeadingElement);
  });
});
