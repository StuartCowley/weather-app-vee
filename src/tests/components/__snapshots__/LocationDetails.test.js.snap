// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocationDetail renders correctly 1`] = `
<DocumentFragment>
  <h1>
    Manchester UK
  </h1>
</DocumentFragment>
`;
